src/PHPStan/GetAttributeCheck.php

```diff
@@ -41,29 +44,29 @@ public function __construct(
      */
     public function check(StaticCall $node, Scope $scope): ?array
     {
-        $errors = [];
-
         $arguments = $this->getNormalizedArguments($node);
 
         if ($arguments === null) {
             return null;
         }
 
+        // TODO: Use `$this->ruleLevelHelper->findTypeToCheck` here.
```

Comment:
The `findTypeToCheck` would remove `null` issue and `unionType` issues.

But I'm not sure it's possible to add an easy `findTypeToCheck` here so maybe I should rely on `reportMaybe` and `reportNull` directly.